lib/misc.py

```diff
@@ -42,7 +42,8 @@ def delete(cli):
 def list_(cli):
    ch.dependencies_check()
    imgdir = ch.storage.unpack_base
-   imgs = ch.ossafe(os.listdir, "can't list directory: %s" % imgdir, imgdir)
+   imgs = ch.ossafe(os.listdir, "not a storage directory: %s" 
+        % ch.storage.root, imgdir)
```

Comment:
I think there's actually two separate error conditions here:

1. `ch.storage.root` does not exist; error should just say that.
2. `ch.storage.root` does exist but doesn't contain a subdirectory `img`. Error should say something to that effect. (I remember seeing a method in `Storage` that's yes/no on whether the storage directory looks valid. Possibly part of PR #1024?)